doc/source/user/basics.dispatch.rst

```diff
@@ -57,7 +57,7 @@ array([[2., 0., 0., 0., 0.],
 Notice that the return type is a standard ``numpy.ndarray``.
 
 >>> type(np.multiply(arr, 2))
-numpy.ndarray
+<class 'numpy.ndarray'>
```

Comment:
note (change is fine): this is Python interpreter, the old value was IPython interpreter. Most users will see the latter, so this is not necessarily helpful, but I don't mind much either way.

Reply:
I didn't realize this. Any version is fine by me.